BTrees/tests/testConflict.py

```diff
@@ -478,7 +504,7 @@ def testConflictWithOneEmptyBucket(self):
 
     @_skip_wo_ZODB
     def testConflictOfInsertAndDeleteOfFirstBucketItem(self):
-        """
+        """ testConflictOfInsertAndDeleteOfFirstBucketItem
```

Comment:
This part doesn't look right, somehow

Actually, our coding standard would be to replace the docstring with a comment (docstrings in testcases mess up the error reporting).